src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -146,6 +146,23 @@ public static class TimeSeriesCatalog
             int windowSize=64, int backAddWindowSize=5, int lookaheadWindowSize=5, int averageingWindowSize=3, int judgementWindowSize=21, double threshold=0.3)
             => new SrCnnAnomalyEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnName, windowSize, backAddWindowSize, lookaheadWindowSize, averageingWindowSize, judgementWindowSize, threshold, inputColumnName);
 
+        /// <summary>
+        /// Create <see cref="DTRootCauseLocalizationEstimator"/>, which localizes root causess using decision tree algorithm.
```

Comment:
>causess [](start = 88, length = 7)

typo

Reply:
Thank you for pointing it out, has fixed it.

Reply:
> > causess [](start = 88, length = 7)
> 
> typo

Thank you for pointing it out, has fixed it.